### ISSUE TITLE
GUI tray / floating widget

### ISSUE THREAD

**Issue by abrichr** [MLDSAI/OpenAdapt#220]
https://github.com/zauberzeug/nicegui/discussions/categories/ideas-feature-requests

**Comment by 0dm** [comment]
made a post about it: https://github.com/zauberzeug/nicegui/discussions/980

**Comment by abrichr** [comment]
@0dm can you please implement? 🙏

**Comment by 0dm** [comment]
@abrichr Yes, I'll get started on it -- hopefully it won't be too tricky..

**Comment by abrichr** [comment]
Closing as part of architecture transition. Legacy frozen at v0.46.0. See PR #960.
